test/v200/utils/api/test_utils.go

```diff
@@ -0,0 +1,173 @@
+package api
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"testing"
+
+	common "github.com/devfile/api/v2/test/v200/utils/common"
```

Comment:
same as https://github.com/devfile/api/pull/348/files#r579330744

Reply:
I plan in my next pull request to make these commonUtils.

Reply:
Fixed in new commit